src/explorer/Queries/EmailCheck.cs

```diff
@@ -0,0 +1,24 @@
+namespace Explorer.Queries
+{
+    using System.Text.Json;
+
+    using Diffix;
+    using Explorer.Common;
+    using Explorer.JsonExtensions;
+
+    internal class EmailCheck :
+        DQuery<long>
+    {
+        public override long ParseRow(ref Utf8JsonReader reader) =>
+            reader.ParseCount();
+
+        protected override string GetQueryStatement(string table, string column)
+        {
+            return $@"
+                select
+                    count(*)
+                from {table}
+                where trim(BOTH '{Constants.EmailAddressChars}' FROM {column}) = '@'";
```

Comment:
Might be worth checking if it isn't more efficient doing something like:

```suggestion
                where right({column}, 4) LIKE '%.%' AND {column} LIKE '%@%'";
```

I.e. basically the intent is to check if:
- it contains a `@`
- it ends in a TLD

At least a quick test seems to suggest the `like` version is about ~3x faster.

It might of course yield false positives, but the chances seem relatively slim?

Reply:
I also did some quick testing, and indeed, the version using `like` is faster. 
I think that there are some computations executed by the explorer which are a little fuzzy, but it is better to have some approximate results rather than no results.